aws-pub-sub/sample.json

```diff
@@ -7,6 +7,9 @@
     "platform": {
         "libs": ["AWS IoT C++ SDK"]
     },
+    "service_provider": "Amazon Web Services (AWS)",
+    "service_provider_url": "https://aws.amazon.com",
```

Comment:
I _think_ I saw you use 'uri' in the code instead of url. Would be good to keep the same string.